website/pages/en/community.js

```diff
@@ -124,7 +124,7 @@ class Index extends React.Component {
                 },
                 {
                   title: 'Sorbet: A type checker for Ruby 3 you can use today!',
-                  link: '/docs/talks/pairsrb-2020',
+                  link: '/docs/talks/parisrb-2020',
```

Comment:
You need to update the filename and the metadata too: https://github.com/sorbet/sorbet/blob/master/website/docs/talks/pairsrb-2020.md